components/BottomNavigation/tests/unit/Theming/MDCBottomNavigationThemingTests.m

```diff
@@ -0,0 +1,180 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MaterialBottomNavigation+Theming.h"
+
+#import <XCTest/XCTest.h>
+
+#import "MaterialColorScheme.h"
+#import "MaterialContainerScheme.h"
+#import "MaterialShadowElevations.h"
+#import "MaterialTypography.h"
```

Comment:
I might have missed it, but I don't see MaterialTypography being used in this test. Can it be removed?

Reply:
Typography is where `mdc_isSimplyEqual` comes from.

Reply:
Ah, yes. Thanks!